test/end-to-end/test/test_createRecipe.js

```diff
@@ -32,6 +32,7 @@ describe('Create Recipe Page', () => {
           .click(recipesPage.btnCreateRecipe)
           .wait(page => document.activeElement.id === page.inputNameEleId
             , createRecipePage)
+          .insert(createRecipePage.inputName, createRecipePage.varEmptyName)
```

Comment:
Hi @jkozol, there're a question about this line of code.
Why do we need to type a empty string in Name input? We don't need to this action for this test case. The reason that I setup this case is to simulate a scenario that a user clicks Save button accidentally without any input.
The change is able to pass this case even though the case does not include this line of code.